src/tests/compiletest.md

```diff
@@ -254,6 +254,17 @@ test based on the debugger currently being used:
   NOTE: The "Rust" version of LLDB doesn't exist anymore, so this will always be ignored.
   This should probably be removed.
 
+> **Note on running lldb debuginfo tests locally**
+>
+> If you want to run lldb debuginfo tests locally, then currently on Windows it is required that:
+> 
+> - You have **exactly** Python 3.10 installed -- not a newer version, not an older version,
```

Comment:
This is a bit misleading. You can have other versions installed but this could be taken as meaning that you have to uninstall like Python 3.12 or whatever.

Reply:
I'll revise this wording slightly.

Reply:
https://github.com/rust-lang/rustc-dev-guide/pull/2076